products/storage/terraform.yaml

```diff
@@ -102,6 +102,35 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         ignore_read: true
   Object: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
+  HmacKey: !ruby/object:Overrides::Terraform::ResourceOverride
+    create_url: projects/{{project}}/hmacKeys?serviceAccountEmail={{service_account_email}}
```

Comment:
Can you override `read_url`, `update_url`, `delete_url` to `projects/{{project}}/hmacKeys/{{access_id}}`? Magic Modules is very helpfully generating `projects/{{project}}/hmacKeys/{{accessId}}`. We really need to support camelcased params better.

Reply:
It looks like those are on the api.yaml not in tf?
`RuntimeError: @read_url does not exist on HmacKey`

Reply:
Ah, apparently we've never actually added that one. I found a similar case where we had to snake_case a value, you can set `self_link` to `projects/{{project}}/hmacKeys/{{accessId}}` instead. The defaults for the `_url` values should come from that.